coralproject/talk
# [CORL-2374]: add For review moderation queue enablement to admin

## What does this PR do?

These changes add the option to enable or disable the "For review" queue in the admin at the bottom of the Advanced configuration panel. They also remove the `FOR_REVIEW` feature flag. These changes are for https://vmproduct.atlassian.net/browse/CORL-2374.

## What changes to the GraphQL/Database Schema does this PR introduce?

These changes add `forReviewQueue` to the settings. They also remove the `FOR_REVIEW` feature flag.

## How do I test this PR?

 Go to the admin --> Configure --> Advanced. At the bottom, you should see a section with the heading "Review all user reports". You can enable/disable the "For review" queue here. Check that if you go to the Moderate tab while it's enabled, you see a "For review" queue. Check that if you go to the Moderate tab while it's disabled, you don't see a "For review" queue.
 
## How do we deploy this PR?

Any customers using this feature will need this option enabled on deploy.
